servers/audio_server.cpp

```diff
@@ -2140,6 +2155,7 @@ void AudioBusLayout::_get_property_list(List<PropertyInfo> *p_list) const {
 		p_list->push_back(PropertyInfo(Variant::BOOL, "bus/" + itos(i) + "/mute", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
 		p_list->push_back(PropertyInfo(Variant::BOOL, "bus/" + itos(i) + "/bypass_fx", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
 		p_list->push_back(PropertyInfo(Variant::FLOAT, "bus/" + itos(i) + "/volume_db", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
+		p_list->push_back(PropertyInfo(Variant::FLOAT, "bus/" + itos(i) + "/volume_linear", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_INTERNAL));
```

Comment:
This should also use `PROPERTY_USAGE_NONE`.

Reply:
Hm, I'll have to look into that again/further since it's been a while since I worked on this - I think I was mostly just copying what looked like relevant code to add my feature 😅 

My idea here, I believe, was to allow the user to access `volume_linear` for things like linearly fading a bus's volume  using a tween.

Reply:
But this property is not exposed, so it's unlikely user would use it. They will use the `set_bus_volume_linear()` method.

Reply:
Ah, yes I see what you mean as I look back at the code again. My hope was to be able to fade the volume out with something like
```gdscript
# Fades out over 1 second
tween.tween_property(audio_bus_layout, "bus/0/volume_linear", 0.0, 1.0)
```
But, looking at my testing code for this feature, I see that you have to save the bus layout *back* to the AudioServer after modifying its properties, which prevents it from being (easily) tweenable as far as I know. Thus, if you don't  think it makes sense to have the `volume_linear` property for AudioBusLayout, I can remove it.

In the future, though, might it make sense to propose a modification to the way that the AudioServer/AudioBusLayout system works, so that a snippet like I provided above would be viable? `set_bus_volume_linear()` may work along with `tween_method` but I think it does have some drawbacks, notably requiring an explicit start value instead of being able to interpolate from whatever the value was when the tween started.

Reply:
> Thus, if you don't think it makes sense to have the volume_linear property for AudioBusLayout, I can remove it.

Yeah remove it. It's not very useful if its not documented anywhere.